client/__tests__/e2e/e2eAnnotations.test.js

```diff
@@ -276,6 +277,26 @@ describe.each([
     await assertLabelDoesNotExist(perTestCategoryName, perTestLabelName);
   });
 
+  test("stacked bar graph renders", async () => {
+    await setup(config);
+
+    await expandCategory(perTestCategoryName);
+
+    await clickOn(`colorby-louvain`);
+
+    const labels = await getAllByClass("categorical-row");
+
+    const result = await Promise.all(
+      labels.map(async (label) => {
```

Comment:
Whoops, I think I forgot to remove `async` keyword here 😆 Please help me remove it! Thank you!